src/components/scheduling/websites/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="bmc-schedule-tab">
-    <create-schedules :content-id="contentId" />
-    <list-schedules :content-id="contentId" />
+    <create-schedules :content-id="contentId" :dateSchedulingEnabled="dateSchedulingEnabled"/>
+    <list-schedules :content-id="contentId" :dateSchedulingEnabled="dateSchedulingEnabled"/>
```

Comment:
Updated.  The conversion between camel and dashed throws me sometimes since it avoids only a possible naming conflict and not a hard requirement and happens on its own.